client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,258 @@
+						</h6>
+						<div className="customer-home__card-button-pair">
+							<Button href={ site.URL } primary>
+								{ translate( 'View Site' ) }
+							</Button>
+							<Button href={ customizeUrl }>{ translate( 'Edit Homepage' ) }</Button>
+						</div>
+					</Card>
+					<Card className="customer-home__card-boxes">
+						<div className="customer-home__boxes">
+							<ActionBox
+								onClick={ () => page( `/page/${ siteSlug }` ) }
+								label={ translate( 'Add a page' ) }
+								iconSrc="/calypso/images/customer-home/page.svg"
+							/>
+							<ActionBox
+								onClick={ () => page( `/post/${ siteSlug }` ) }
+								label={ translate( 'Write blog post' ) }
+								iconSrc="/calypso/images/customer-home/post.svg"
+							/>
+							<ActionBox
+								href={ customizeUrl }
+								label={ translate( 'Customize theme' ) }
+								iconSrc="/calypso/images/customer-home/customize.svg"
+							/>
+							<ActionBox
+								onClick={ () => page( `/themes/${ siteSlug }` ) }
+								label={ translate( 'Change theme' ) }
+								iconSrc="/calypso/images/customer-home/theme.svg"
+							/>
+							<ActionBox
+								href={ customizeUrl }
+								label={ translate( 'Edit menus' ) }
+								iconSrc="/calypso/images/customer-home/menus.svg"
+							/>
+							<ActionBox
+								href="https://support.wordpress.com/images/"
+								label={ translate( 'Change images' ) }
+								iconSrc="/calypso/images/customer-home/images.svg"
+							/>
+							<ActionBox
+								href="https://logojoy.grsm.io/looka"
+								label={ translate( 'Design a logo' ) }
+								iconSrc="/calypso/images/customer-home/logo.svg"
+							/>
+							<ActionBox
+								href={ `https://wordpress.com/email/${ siteSlug }` }
+								label={ translate( 'Add G Suite' ) }
+								iconSrc="/calypso/images/customer-home/gsuite.svg"
+							/>
+						</div>
+					</Card>
+				</div>
+				<div className="customer-home__layout-col">
+					<Card>
+						<CardHeading>{ translate( 'Grow & Earn' ) }</CardHeading>
+						<h6 className="customer-home__card-subheader">
+							{ translate( 'Grow my audience and earn money' ) }
+						</h6>
+						<VerticalNav className="customer-home__card-links">
+							<VerticalNavItem path={ `/marketing/connections/${ siteSlug }` }>
+								{ translate( 'Share my site' ) }
+							</VerticalNavItem>
+							<VerticalNavItem path={ `/marketing/tools/${ siteSlug }` }>
+								{ translate( 'Grow my audience' ) }
+							</VerticalNavItem>
+							<VerticalNavItem path={ `/earn/${ siteSlug }` }>
+								{ translate( 'Earn money' ) }
+							</VerticalNavItem>
+						</VerticalNav>
+					</Card>
+					<Card>
+						<CardHeading>{ translate( 'Support' ) }</CardHeading>
+						<h6 className="customer-home__card-subheader">
+							{ translate( 'Get all of the help you need' ) }
+						</h6>
+						<div className="customer-home__card-support">
+							<img
+								src="/calypso/images/customer-home/happiness.png"
```

Comment:
This image seems a bit odd for three reason:

- It seems really blurry compared to other images in Calypso
- The colours vary from the rest of the ones in Calypso (#30248)

Those could be intentional, but this feels like a bug:
- The proportions are off on my tablet

![Screenshot_20190812-220957](https://user-images.githubusercontent.com/43215253/62898781-3bce8600-bd4e-11e9-99ac-7b611c963e67.jpg)

Reply:
Interesting! Can you provide more info about the device where you're experiencing this stretching?

Reply:
Samsung Galaxy Tab S2 (Android 7) :)